src/main/java/edu/ucla/library/prl/harvester/Institution.java

```diff
@@ -352,4 +352,43 @@ public URL getWebsite() {
     public static Institution withID(final Institution anInstitution, final Integer anInstitutionID) {
         return new Institution(anInstitution.toJson().put(ID, anInstitutionID));
     }
+
+    @Override
+    @SuppressWarnings("PMD.CyclomaticComplexity")
+    public boolean equals(final Object anOther) {
+        if (anOther instanceof Institution) {
+            final Institution other = (Institution) anOther;
+
+            if (getID().equals(other.getID()) && getName().equals(other.getName()) &&
```

Comment:
Any opinion on trimming whitespace in the string fields, to eliminate typos that might produce a false negative? Or trimming strings before add/update operations? Would also apply to Job.java

Reply:
Sure, I could see that happening during construction. I would prefer that as a separate PR though.

Reply:
Create a new ticket for it?

Reply:
SERV-791